app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState, useEffect } from 'react';
+
+const Greeting = () => {
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    fetch('/random_greeting')
+      .then(response => response.json())
+      .then(data => setMessage(data.message));
+  }, []);
+
+  return <h2>{message}</h2>;
+};
```

Comment:
- [ ] @MNisarAli Great work adding Greeting component, however the component is supposed to use data from our redux store to display a random greeting and not fetch the data directly, kindly consider refactoring your component
![image](https://github.com/MNisarAli/hello-rails-react/assets/61408860/2a59acc8-cfbc-4990-b706-fd8ed6176d40)